### DISCUSSION TITLE
Why are there differnt tags for differnt architectures?

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#675] [Questions] [answered]
I just wanted to ask why there are different tags for the different architectures, since docker image can have the same tag and can be used for multiple architectures?

**Comment by szaimen** [comment, marked as answer]
Hello, see https://github.com/nextcloud/all-in-one/discussions/490

**Comment by Zoey2936** [comment]
So, do I understand correct, the problem is not pulling or running these images? It is that the update checking is not possible, since docker hub always report the checksum of the amd64 build, which would be always different from the checksum of the arm64 build and with that it will ever say that there is a new version? Or did is misunderstood the issue with one tag for both architectures?

**Reply by szaimen** [reply]
> It is that the update checking is not possible, since docker hub always report the checksum of the amd64 build, which would be always different from the checksum of the arm64 build and with that it will ever say that there is a new version?

yes, this is the reason 👍

**Comment by Zoey2936** [comment]
I don't know if and how this would be possible and how makeable it is, but why not comparing the dates, the local images got created and the date the images on docker hub got created? The dates would always be the same.

**Reply by szaimen** [reply]
Honestly, changing this now will not allow us to get rid of the arm64 containers because many people already run it with these and it is not possible to change this for already existing instances, unfortunatley... I see the point of multiarch images but I don't think it is that bad to have different names for the different architectures...

**Reply by Zoey2936** [reply]
Images could still be pushed onto the arm tags, so that they exist, but are also multi arch, like the normal tags, this would not be a disadvantage in my opinion, if the check over the date would work. But I understand that multi arch images do not bring big advantages and this change would take some work, if it all even would work. It was just an Idea how this could be done.

**Reply by szaimen** [reply]
All right. Thanks for the brainstorming and bringing in some ideas on this! :)
